index.md

```diff
@@ -271,12 +271,37 @@ The heart of the Ruby Unconference is people. [This guideline](https://berlincod
 Read the full [Berlin Code of Conduct here](https://berlincodeofconduct.org/).
 </div>
 
+<div class="content-section" markdown="1">
+## Team
+### Organizer
+
+  <ul class="avatar__list">
+    <li class="avatar avatar-purple"><img class="avatar-image" src="/assets/images/team/sl.jpg"></li>
+    <li class="avatar avatar-orange"><img class="avatar-image" src="https://pbs.twimg.com/profile_images/822309015/moi_klein_160_400x400.jpg"></li>
+    <li class="avatar avatar-purple"><img class="avatar-image" src="/assets/images/team/jz.jpg"></li>
+    <li class="avatar avatar-orange"><img class="avatar-image" src="/assets/images/team/sk.jpg"></li>
+    <li class="avatar avatar-purple"><img class="avatar-image" src="/assets/images/team/asd.jpg"></li>
+    <li class="avatar avatar-orange"><img class="avatar-image" src="/assets/images/team/il.jpg"></li>
+    <li class="avatar avatar-purple"><img class="avatar-image" src="https://pbs.twimg.com/profile_images/1609588568/eightbit__1__400x400.png"></li>
+    <li class="avatar avatar-orange"><img class="avatar-image" src="/assets/images/team/asd.jpg"></li>
+    <li class="avatar avatar-purple"><img class="avatar-image" src="https://pbs.twimg.com/profile_images/1131088445600948224/y57hJmqn_400x400.png"></li>
+  </ul>
+
+### MCs
+
+  <ul class="avatar__list">
+    <li class="avatar avatar-purple"><img class="avatar-image" src="/assets/images/team/jk.jpg"></li>
+    <li class="avatar avatar-orange"><img class="avatar-image" src="https://pbs.twimg.com/profile_images/824642411750166528/rvoehSNV_400x400.jpg"></li>
+    <li class="avatar avatar-purple"><img class="avatar-image" src="/assets/images/team/om.jpg"></li>
+    <li class="avatar avatar-orange"><img class="avatar-image" src="https://pbs.twimg.com/profile_images/2173945060/image_400x400.jpg"></li>
+    <li class="avatar avatar-purple"><img class="avatar-image" src="https://pbs.twimg.com/profile_images/2173945060/image_400x400.jpg"></li>
```

Comment:
Uygar appears twice in the list and Iryna is missing.

Reply:
@lukad This is because of I don't have all images yet. Thanks for the heads up.

Reply:
Ok, I thought this might have been a copy paste mistake.